k8s/overlays/prod/patches/invoice-cronjob.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: invoice-cronjob
+spec:
+  schedule: "0 13 1 * 0"
```

Comment:
You should remove this schedule here otherwise it'll override what you define in base.